Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -105,6 +106,8 @@ void TextEditor::create_actions()
             this);
     }
     m_select_all_action = CommonActions::make_select_all_action([this](auto&) { select_all(); }, this);
+    m_insert_emoji_action = CommonActions::make_insert_emoji_action([&](auto&) { insert_emoji(); },
+        this);
```

Comment:
Why `this` on a new line?

Reply:
That was formatting by CLion before I moved the contents of `insert_emoji()` into its own function, my bad.